devpi_ldap/main.py

```diff
@@ -175,7 +175,7 @@ def _search(self, conn, config, **kw):
             if any(attribute_name in x.get('attributes', {}) for x in conn.response):
                 def extract_search(s):
                     if 'attributes' in s:
-                        return s['attributes'][attribute_name]
+                        return [s['attributes'][attribute_name]]
```

Comment:
This is one of those painful but useful changes. You will probably find that you want to pin to ldap3 < 2, then adapt the code to support and require ldap3 > 2. This is the approach we had to take as there's no compatibility layer bridging the difference in behavior.